ouroboros-network-api/src/Ouroboros/Network/PeerSelection/RelayAccessPoint.hs

```diff
@@ -53,6 +58,49 @@ data RelayAccessPoint = RelayAccessDomain  !DNS.Domain !Socket.PortNumber
                       | RelayAccessAddress !IP.IP      !Socket.PortNumber
   deriving (Eq, Ord)
 
+-- | These instances are used to serialize 'LedgerPeerSnapshot'
+-- consensus LocalStateQuery server which uses these instances
+-- for all its query responses. It appears they provide some improved
+-- debugging diagnostics over Serialize instances.
+instance ToCBOR RelayAccessPoint where
```

Comment:
Do we have a round trip tests for `{To,From}CBOR` instances?

Reply:
Yes.